src/Components/CatProfile.tsx

```diff
@@ -6,28 +6,33 @@ import FormElement from './Styling/FormElement';
 import Button from './Styling/Button';
 import { Cat } from '../api';
 
+const loggedInUser = 'loggedInUser';
```

Comment:
Should be `USER1` or some other user in the dev database.

Reply:
I just checked the current seed data, and we're currently seeding with `user1` in lowercase.

Reply:
The version I can see still says `USER1`?